src/shadowbox/scripts/update_mmdb.sh

```diff
@@ -9,11 +9,11 @@ TMPDIR="$(mktemp -d)"
 FILENAME="ip-country.mmdb"
 
 # We need to make sure that we grab an existing database at install-time
-for monthdelta in {0..10}; do
-    newdate=$(date --date="-$monthdelta month" +%Y-%m)
-    ADDRESS="https://download.db-ip.com/free/ip-country-${newdate}.mmdb.gz"
-    curl --fail --silent "${ADDRESS}" -o "$TMPDIR/$FILENAME.gz" > /dev/null && break
-    if (( monthdelta == 10 )); then
+for monthdelta in 0 1 2 3 4 5 6 7 8 9 10; do
+    newdate=$(date --date="-$monthdelta months" +%Y-%m)
+    ADDRESS="https://download.db-ip.com/free/dbip-country-lite-${newdate}.mmdb.gz"
+    curl --fail --silent "${ADDRESS}" -o "$TMPDIR/$FILENAME.gz" > /dev/null && break  
+    if [[ $monthdelta -eq 10 ]]; then
```

Comment:
Revert

Reply:
(( doesn't work on dash.  If I revert I get an error: "monthdelta not found"